tools/cldr-code/src/main/java/org/unicode/cldr/util/SupplementalDataInfo.java

```diff
@@ -2794,7 +2799,11 @@ private Set<String> getTargetTerritories(String language) {
             // fall through
         }
         if (targetTerritories.size() == 0) {
-            targetTerritories.add("ZZ");
+            if (secondaryTerritories.size() > 0) {
+                targetTerritories.addAll(secondaryTerritories);
```

Comment:
Looks better, but this could add multiple secondary territories when we only really want the top one. Best would be to use LikelySubtags first. Something like:

```
    if (targetTerritories.size() == 0) {
        String region = null;
        String maximized = new LikelySubtags().maximize("raj");
        if (maximized != null) {
            CLDRLocale cldrLocale = CLDRLocale.getInstance(maximized);
            region = cldrLocale.getCountry();
        }
        if (region != null) {
            targetTerritories.addAll(region);
        } else if (secondaryTerritories.size() > 0) {
            targetTerritories.addAll(secondaryTerritories);
```